elfpy/bots/create_default_profile.py

```diff
@@ -1,9 +1,9 @@
 """Example script for writing a custom bot config to json"""
-from elfpy.bots import DEFAULT_USERNAME, BotConfig
+from elfpy.bots import DEFAULT_USERNAME, EnvironmentConfig
```

Comment:
My assumption was that this file is now obsolete in favor of `config.py` in examples/eth_bots

Reply:
@dpaiton is this right?

Reply:
we replaced BotConfig with EnvironmentConfig in #666 and I haven't seen anything to indicate otherwise since

Reply:
This is just through looking at and running `main.py` in `eth_bots`. I guess this makes sense if we still want to support `evm_bots` through this patch